python/py-smbus2/Portfile

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-smbus2
+version             0.4.3
+revision            0
+categories          python
+maintainers         nomaintainer
+platforms           any
```

Comment:
add `darwin`?

Reply:
As in https://github.com/macports/macports-ports/pull/20639#discussion_r1341875995—I don’t see a `.so` for this Python module.